R/as.network.numeric.R

```diff
@@ -65,6 +65,25 @@ as.network.numeric<-function(x,
     ignore.eval = TRUE, names.eval = NULL,
     edge.check = FALSE,
     density=NULL, init=NULL, numedges=NULL, ...){
+  # Producing an informative error for each of the following invalid or unsupported inputs
+  if(loops == TRUE)
+    stop("Loop can't be contained in your random network")
+  if(multiple == TRUE)
+    stop("Multiple edges can't be contained in your random network")
+  if(hyper == TRUE)
+    stop("Hyper edges can't be contained in your random network")
```

Comment:
Please rephrase as "Generating networks with X is not supported at this time."

Also, instead of "multiple edges" write "multigraphs".